images/channels/vidi-space.svg

```diff
@@ -0,0 +1,736 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
This is a "fake" svg - it's actually just an embedded png. Please request a vector format image file from the channel.

Reply:
This has been corrected. VIDI SPACE sent over a new .svg, I ran it through minifier and uploaded.